sopel/modules/url.py

```diff
@@ -231,6 +259,18 @@ def get_hostname(url):
     return hostname
 
 
+def get_tinyurl(url):
+    """ Returns a shortened tinyURL link of the URL. """
+    tinyurl = "https://tinyurl.com/api-create.php?url=%s" % url
+    try:
+        res = requests.get(tinyurl)
+    except requests.exceptions.ConnectionError:
```

Comment:
The `ConnectionError` exception won't cover all cases. Timeouts, misconfigurations that cause excessive redirects, invalid/malformed HTTP responses… these all have different exceptions in Requests.

But! We can catch them all by catching the base class:

```suggestion
    except requests.exceptions.RequestException:
```